higlass/client.py

```diff
@@ -143,9 +150,7 @@ def change_attributes(self, **kwargs):
         """
         conf = self.conf.copy()
         conf.update(kwargs)
-        return self.__class__(
-            conf["type"], position=self.position, tileset=self.tileset, **conf
-        )
+        return self.__class__(conf["type"], position=self.position, tileset=self.tileset, **conf)
```

Comment:
Formatting 😐

Reply:
Is this personal flavor or does it come from black?

Reply:
Black.